src/pages/home/sidebar/SidebarScreen/index.js

```diff
@@ -0,0 +1,40 @@
+import React, {PureComponent} from 'react';
+import BaseSidebarScreen from './BaseSidebarScreen';
+
+class SidebarScreen extends PureComponent {
```

Comment:
I think we can change this `Function` component. We're not using `setState` and hence don't need this to be a `Class Component`